spec/controllers/two_factor_authentication/webauthn_verification_controller_spec.rb

```diff
@@ -221,7 +221,8 @@
 
           it 'redirects to webauthn error page ' do
```

Comment:
Since we've removed the logic branch for handling "only has WebAuthn as an MFA method", I think we could remove this entire `context` block for 'User only has webauthn as an MFA method' from lines 215 to 230, since we're already checking the base redirect behavior above on line 182.

(Arguably, we could also flatten / unwrap the context from line 150 to 213, for the same reason)

Reply:
Those changes are applied to the spec. Thanks!